core/string/uminus_spec.rb

```diff
@@ -69,4 +69,11 @@
       (-dynamic).should equal(-"this string is frozen".freeze)
     end
   end
+
+  ruby_version_is "3.0" do
```

Comment:
I thought this was 2.7+ behavior, but it's actually 2.8/3.0.

Any reason why CI doesn't test against ruby-head?

Reply:
> Any reason why CI doesn't test against ruby-head?

Because ruby/ruby changes the specs and so it would fail except when specs are just synchronized.

So for specs for not-yet-released CRuby versions, one way to test is to make a PR against ruby/ruby.

Reply:
https://github.com/ruby/spec#synchronization-with-ruby-implementations aims to explain that